doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -10,9 +10,54 @@
 
 <a name="6.1"></a>
 ## 6.1 Introduction
+The adoption of cloud-native allows development teams to move fast, deploy software efficiently, and operate scale. 
+Cloud-native applications offer the benefits by providing applications as small microservices (either containers or serverless), and managing them through DevOps processes. To build these applications, organisations need a cloud-native infrastructure which provides flexibility and enough services for developers. Cloud-native infrastructures should provide a platform for the build and release team to effectively perform continuous delivery of its applications, with close to zero human intervention. With Cloud-native, the industy requires a radically different apporach to security in contrast to monolithic architecture. 
 
 <a name="6.2"></a>
-## 6.2 Heading
+##  6.2 Principles
+### Kubernetes Security
+The following are core principles to consider when securing cloud-native applications and infrastructure;
+
+- Deploy only secure applications and trusted codes
+- Only deploy applications from validated and verified images
+- Only deploy applications from trusted registries
+- Containers orchestration (Kubernetes) must be secure with administrative boundaries between tenants
+  - Use Namespaces to establish security boundaries between tenants
+  - Create and define cluster network policies
+  - Run a cluster-wide pod security policy
+  - Turn on Audit Logging
+  - Separate sensitive workloads using namespaces
+  - Secure tenant metadata Access
+- Network segmentation using security zoning and network standards must be followed when deploying containers 
+- The Host OS running the containers must be hardened
+- Use container-aware runtime defense tools
+- Enable Role-Based Access Control (RBAC)
+
+##  6.3 Kubernetes Security Architecture
+Kubernetes Security architecture is made up of of several components that provide end to end security for the infrastructure and application or workload. These components include image registries (private or public), Kubernetes API, RBAC, network services, host and container runtime, keys or secret management. Each of these components work in a collaborative mode to ensure the security of the applications or workload. 
 
 <a name="6.3"></a>
-## 6.3 Heading
+## 6.4 Security Perimeters
```

Comment:
Are the definitions necessary in this section?

Reply:
I have removed long definitions and and re-phrased the points in Section 6.4.
@tomkivlin Pls take a look again at this section.

Reply:
resolved

Reply:
@msidana looking good 👍